module/platform/NUSense/HardwareIO/src/HardwareIO.cpp

```diff
@@ -119,6 +119,9 @@ namespace module::platform::NUSense {
             // Battery data
             sensors->battery = 0;  // not yet implemented
 
+            // Log the latest dispatch from NUSense; only very rare exceptions should be dispatched.
+            log<NUClear::WARN>("Dispatch from NUSense: " + data.dispatch);
```

Comment:
Would this be better as `DEBUG` instead?

Reply:
WARN or INFO is best, unless you want to specify log level on the NUsense side and display accordingly here. WARN is probably what we want though for anything coming from subcontroller